plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/database/DatabaseNavigatorLabelProvider.java

```diff
@@ -187,32 +191,40 @@ public String getToolTipText(Object element) {
             final DBPDataSourceContainer ds = ((DBNDataSource) element).getDataSourceContainer();
             if (ds != null) {
                 StringBuilder info = new StringBuilder();
```

Comment:
Just a tip: this can be replaced with the `StringJoiner`, so we don't need to manually append `\n` and then trim the final string.